src/client.ts

```diff
@@ -502,7 +502,7 @@ export interface IStartClientOpts {
     /**
```

Comment:
These are all good changes, but I suggest removing them from this PR and putting them into a separate one just to make it easier to review.